tests/unit/amo/pages/TestUsersUnsubscribe.js

```diff
@@ -166,32 +149,24 @@ describe(__filename, () => {
       const { store } = dispatchClientMetadata();
       _finishUnsubscribeNotification(store, params);
 
-      const root = render({ params, store });
+      render({ params, store });
 
-      expect(root.find('.UsersUnsubscribe-content-notification')).toHaveText(
-        getNotificationDescription(fakeI18n(), notificationName),
-      );
+      expect(
+        screen.getByText(
+          getNotificationDescription(fakeI18n(), notificationName),
+        ),
+      ).toBeInTheDocument();
     });
 
     it('renders a link to edit the user profile', () => {
       const { store } = dispatchClientMetadata();
       _finishUnsubscribeNotification(store);
 
-      const root = render({ store });
+      render({ store });
 
-      expect(
-        root.find('.UsersUnsubscribe-content-edit-profile').childAt(0),
-      ).toHaveText('You can edit your notification settings by ');
-      // The second child is a `Link`.
-      expect(
-        root.find('.UsersUnsubscribe-content-edit-profile').childAt(1),
-      ).toHaveProp('to', '/users/edit');
-      expect(
-        root.find('.UsersUnsubscribe-content-edit-profile').childAt(1),
-      ).toHaveProp('children', 'editing your profile');
-      expect(
-        root.find('.UsersUnsubscribe-content-edit-profile').childAt(2),
-      ).toHaveText('.');
+      const link = screen.getByRole('link', { name: 'editing your profile' });
+      expect(link).toBeInTheDocument();
```

Comment:
We don't need this. The previous statement would fail if it wasn't found, as would the subsequent one. You could combine all three into one statement:

`expect(screen.getByRole('link', { name: 'editing your profile' })).toHaveAttribute('href', '/en-US/android/users/edit');`